lib/declare_schema/model/column.rb

```diff
@@ -132,7 +132,7 @@ def collation_and_charset_for_column(current_table_name, column_name)
         database_name = connection.current_database
 
         defaults = connection.select_one(<<~EOS)
-          SELECT C.character_set_name, C.collation_name
+          SELECT C.character_set_name as character_set_name, C.collation_name as collation_name
```

Comment:
@ColinDKelley Tacking on an additional bug fix that @cmparkinson found when playing with MySQL 8. Without the `as` aliasing of the column names, his MySQL configuration was returning these fields in their ALL_CAPS form. Now we're safe in both places we're querying `information_schema`.

Reply:
Interesting! That's really odd. Glad you were able to get the fix in here.